# CropLine.msg

float32 m
float32 b